src/main/kotlin/com/nikitanazarov/coroutinestacks/CoroutineTraceForestBuilder.kt

```diff
@@ -77,10 +74,13 @@ private fun SuspendContextImpl.createCoroutineTraceForest(
         return null
     }
     val averagePreferredWidth = maxWidth / traceNotNullCount
-    initialWidth = averagePreferredWidth
+    val averagePreferredCellHeight = vertexData.firstOrNull()!!.preferredHeight / vertexData.firstOrNull()!!.model.size
```

Comment:
You can initialise these variables inside the `forEach` loop and you won't have to use the `!!` operator.
Like:
```
var averagePreferredCellHeight: Int? = null
var fontSize: Int? = null
...
traces.forEach { trace ->
...
averagePreferredCellHeight = ...
fontSize = ...
...
}

if (averagePreferredCellHeight == null || fontSize == null) {
  return
}
```